mobile/src/stateMachine/views/main/home.tsx

```diff
@@ -399,33 +406,39 @@ export class HomeView extends ViewState<{ opacity: Animated.Value, isUnfinishedQ
-                            </TouchableNativeFeedback>
-                        }
-                    </Portal>
-                    {/* MK-TODO below buttons used for development/testing only and will be removed */}
-                    {/* <View style={{ flexDirection: 'row' }}>
-                        <Button title='TESTING' style={styles.testingButton} onPress={() => this.onTESTINGButton()} theme={this.theme} />
-                    </View> */}
-                    {this.state.isUnfinishedQol === null ? <Text>Loading..</Text> : this.getCenterElement()}
-                    {loading
-                        ? <ActivityIndicator size='large' />
-                        : this.getResourcesList()
-                    }
-                    <BottomBar screen={'home'} theme={this.theme} />
-                </Animated.View>
+                {this.userNeedsToUpdateApp ?
+                    <Text style={[this.textStyles.labelExtraLarge, { textAlign: 'center', color: this.theme.colors.foreground, marginHorizontal: 40, marginBottom: 200 }]}>Please update the PolarUs App</Text>
+                    :
+                    <>
+                        <View style={{ position: 'absolute', top: 0, left: 0, right: 0 }}>
+                            <Text style={[this.textStyles.labelMedium, { textAlign: 'center', color: this.theme.colors.midground }]}>{formatDateDayMonthYear(new Date())}</Text>
+                        </View>
+                        <Animated.View style={[this.baseStyles.container, styles.container, { height: this._contentHeight, opacity: this.state.opacity }]}>
+                            {/* Portal component used to capture touch events on or above orb */}
+                            <Portal>
+                                {Platform.OS == 'ios' ?
+                                    <Pressable onPress={this.onTapOrb} style={{ width: Layout.window.width, height: this.orbTapContainerHeight, zIndex: 9999 }}>
+                                        <View style={{ width: '100%', height: this.orbTapContainerHeight }} />
+                                    </Pressable>
+                                    :
+                                    <TouchableNativeFeedback onPress={this.onTapOrb} style={{ width: Layout.window.width, height: this.orbTapContainerHeight, zIndex: 9999 }}>
+                                        <View style={{ width: '100%', height: this.orbTapContainerHeight }} />
+                                    </TouchableNativeFeedback>
+                                }
+                            </Portal>
+                            {/* MK-TODO below buttons used for development/testing only and will be removed */}
+                            {/* <View style={{ flexDirection: 'row' }}>
+                    <Button title='TESTING' style={styles.testingButton} onPress={() => this.onTESTINGButton()} theme={this.theme} />
+                </View> */}
```

Comment:
Can we remove this now? Or are we still using it?

Reply:
nope ill remove